### PR TITLE
Stage 2.33.0

### PR DESCRIPTION
**Pull request by jpogran**
- **Bump terraform-ls from 0.35.0-beta2 to 0.35.0**
- **Stage 2.33.0 release**

### CODE REVIEW

File: .changes/v2.33.0.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+## 2.33.0 (2024-10-14)
+
+NOTES:
+
+This release adds support for Terraform Stack and Deployment files. This provides intelligent completion for Terraform Stacks blocks and attributes in Stack and Deployment files, including suggesting only valid completions for variables in component and deplyoment blocks. Hover documentation for all Stack and Deploy blocks, with helpful type and usage information. Early Validation has been extended to produce diagnostics along with syntax validation for Stack and Deployment files. Reference support allows code navigation in and between Stack and Deploy files. This also provides formatting support for both Stack and Deploy files.
```
Comment (ansgarm) [inline .changes/v2.33.0.md:5]:
```suggestion
This release adds support for Terraform Stack and Deployment files. This provides intelligent completion for Terraform Stacks blocks and attributes in Stack and Deployment files, including suggesting only valid completions for variables in component and deployment blocks. Hover documentation for all Stack and Deploy blocks, with helpful type and usage information. Early Validation has been extended to produce diagnostics along with syntax validation for Stack and Deployment files. Reference support allows code navigation in and between Stack and Deploy files. This also provides formatting support for both Stack and Deploy files.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -80,6 +80,74 @@ INTERNAL:
 * Add development launch target for extensionHost ([#1805](https://github.com/hashicorp/vscode-terraform/issues/1805))
 * Replace webpack with esbuild ([#1807](https://github.com/hashicorp/vscode-terraform/issues/1807))
 
+## 2.33.0 (2024-10-08)
+
+NOTES:
+
+This release adds support for Terraform Stack and Deployment files. This provides intelligent completion for Terraform Stacks blocks and attributes in Stack and Deployment files, including suggesting only valid completions for variables in component and deplyoment blocks. Hover documentation for all Stack and Deploy blocks, with helpful type and usage information. Early Validation has been extended to produce diagnostics along with syntax validation for Stack and Deployment files. Reference support allows code navigation in and between Stack and Deploy files. This also provides formatting support for both Stack and Deploy files.
```
Comment (ansgarm) [inline CHANGELOG.md:87]:
```suggestion
This release adds support for Terraform Stack and Deployment files. This provides intelligent completion for Terraform Stacks blocks and attributes in Stack and Deployment files, including suggesting only valid completions for variables in component and deployment blocks. Hover documentation for all Stack and Deploy blocks, with helpful type and usage information. Early Validation has been extended to produce diagnostics along with syntax validation for Stack and Deployment files. Reference support allows code navigation in and between Stack and Deploy files. This also provides formatting support for both Stack and Deploy files.
```

File: package.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "name": "terraform",
   "displayName": "HashiCorp Terraform",
   "description": "Syntax highlighting and autocompletion for Terraform",
-  "version": "2.33.2024090609",
+  "version": "2.33.0",
```
Comment (ansgarm) [inline package.json:6]: I'm wondering whether we need to change this to 2.34.0 or release a new pre-release immediately after for pre-release folks to get the latest changes 🤔

**Review by ansgarm — COMMENTED** [review]
Looks good for the most parts! Just got a question since I forgot since we last released 😄

File: package.json (hunk 3 above)
Comment (jpogran) [reply package.json:6]: We release a new pre-release immediately after

**Review by ansgarm — APPROVED** [review]

**Comment by jpogran** [comment]
This is expected to fail in CI because we haven't released yet.

**Pull request merged**